doc/pages/administrator/050-handling_certificates.md

```diff
@@ -0,0 +1,267 @@
+
+The two alternative procedures for securing your Astarte deployment are outlined in the following
+sections.
+
+## Use your own certificates
+
+If you already own certificates for your domains, all it's needed is creating a TLS secret in the
+namespace in which Astarte resides. Assuming that the certificate and key are saved respectively as
+`cert.pem` and `privkey.pem`, simply run:
+
+```bash
+$ kubectl create secret tls astarte-tls-cert -n astarte \
+  --cert=cert.pem --key=privkey.pem
+```
+
+## Use Let's Encrypt certificates with cert-manager
+
+The process of obtaining a TLS certificate from Let's Encrypt is handled by cert manager using a
+cluster issuer. The issuer will query the Let's Encrypt API and handles the challenge to confirm
+that you are the right owner of the specified domain. Two types of challenges are supported, namely
+DNS01 and HTTP01.
+
+Ensure all the [prerequisites](020-prerequisites.html) are satisfied and that both cert-manager and
+the NGINX ingress controller are deployed within your cluster. If you haven't installed them yet,
+you can do it with these simple commands:
+
+- install cert-manager:
+```bash
+$ helm repo add jetstack https://charts.jetstack.io
+$ helm repo update
+$ kubectl create namespace cert-manager
+$ helm install \
+  cert-manager jetstack/cert-manager \
+  --namespace cert-manager \
+  --version v1.1.0 \
+  --set installCRDs=true
+```
+- install NGINX ingress controller:
+```bash
+$ helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
+$ helm repo update
+$ helm install ingress-nginx ingress-nginx/ingress-nginx -n ingress-nginx \
+    --set controller.service.externalTrafficPolicy=Local \
+    --create-namespace
+```
+
+### HTTP01 Challenge
+
+The current section outlines the procedure for setting up a ClusterIssuer to solve the HTTP01
+challenge.
+
+#### Define a DNS Zone for your project
+
+First, ensure that a **DNS Zone is already defined for your project**. If this requirement is not
+satisfied, this [page](https://cloud.google.com/dns/docs/zones) provides guidance for the creation
+of the DNS Zone for a project hosted on Google Cloud. If your cluster is hosted by any other cloud
+provider, please ensure to follow the needed steps to fulfill the requirement.
+
+#### Find the external IP assigned to the ingress controller
+
+Knowing the external IP of the NGINX ingress controller is crucial for solving the HTTP01 challenge.
+You can find the external IP under the `EXTERNAL-IP` field when inspecting the output of the
+following command:
+
+```bash
+$ kubectl get svc -n ingress-nginx ingress-nginx-controller
+```
+
+#### Update the DNS Zone
+
+Once the external IP of the ingress controller is known, update your DNS Zone setting the retrieved
+IP for each of the following record sets:
+
+- `api.your-domain.example.com`
+- `dashboard.your-domain.example.com` (if deployed)
+- `broker.your-domain.example.com`
```

Comment:
You don't need a DNS Zone, you just require that the 3 domains all point to the right IP. For all we care they could be in 3 different domains handled by 3 different domain name providers.

For the sake of brevity i would merge `Define a DNS Zone for your project` and `Update the DNS Zone` into a single paragraph
```
#### Configure your DNS
Once the external IP of the ingress controller is known, make sure all your Astarte domains point to the NGINX Ingress controller IP

- `api.your-domain.example.com`
- `dashboard.your-domain.example.com` (if deployed)
- `broker.your-domain.example.com`
```